cmd/cb-event-forwarder/main.go

```diff
@@ -294,6 +310,12 @@ func outputMessage(msg map[string]interface{}) error {
 	return nil
 }
 
+func splitDelivery(deliveries <-chan amqp.Delivery, messages chan<- amqp.Delivery) {
```

Comment:
Is this going to really change the performance of the forwarder at all? Or just help us get a better idea of it? While moving buffering into code we control is nice, it doest really help us meet the output rate > input rate problems.